content/influxdb/v2.0/reference/flux/stdlib/monitor/notify.md

```diff
@@ -29,26 +29,51 @@ monitor.notify(
 ## Parameters
 
 ### endpoint
-A function that constructs and sends the notification to an endpoint.
+({{< req >}}) A function that constructs and sends the notification to an endpoint.
 
 _**Data type:** Function_
 
 ### data
-Data to append to the output.
-**InfluxDB populates notification data.**
+({{< req >}}) Notification data to append to the output.
+This data specifies which notification rule and notification endpoint to associate
+with the sent notification.
 
 _**Data type:** Record_
 
+The data record must contain the following fields:
+
+- _notification_rule_id
+- _notification_rule_name
+- _notification_endpoint_id
+- _notification_endpoint_name
+
 ## Examples
 
 ### Send a notification to Slack
 ```js
 import "influxdata/influxdb/monitor"
+import "influxdata/influxdb/secrets"
 import "slack"
 
-endpoint = slack.endpoint(name: "slack", channel: "#flux")
+token = secrets.get(key: "SLACK_TOKEN")
+
+endpoint = slack.endpoint(token: token)(mapFn: (r) => ({
+    channel: "Alerts",
+    text: r._message,
+    color: "danger"
+  }))()
+
+notification_data = {
+	_notification_rule_id: "0000000000000001",
+	_notification_rule_name: "example-rule-name",
+	_notification_endpoint_id: "0000000000000002",
+	_notification_endpoint_name: "example-endpoint-name",
+}
 
 from(bucket: "system")
 	|> range(start: -5m)
-	|> monitor.notify(endpoint: endpoint)
+	|> monitor.notify(
+    endpoint: endpoint,
+    data: notification
```

Comment:
I think this should be `notification_data`, given the assignment statement above.

Reply:
Yep, good catch